org.osgi.service.feature/src/org/osgi/service/feature/FeatureService.java

```diff
@@ -33,21 +33,67 @@
      * Get a factory which can be used to build feature model entities.
      * @return A builder factory.
      */
-    public BuilderFactory getBuilderFactory();
+	BuilderFactory getBuilderFactory();
+
+	/**
+	 * Obtain an ID from a Maven Coordinates formatted string. The supported
+	 * syntax is as follows:
+	 * <p>
+	 * {@code groupId ':' artifactId ( ':' type ( ':' classifier )? )? ':' version }
+	 * 
+	 * @param coordinates The Maven Coordinates.
+	 * @return the ID.
+	 */
+	ID getIDfromMavenID(String coordinates);
```

Comment:
The method name is still using "MavenID". I assumed @stbischof was commenting on the method name not just the parameter name. This method should be another overload  of `getID`.

Reply:
Sorry guys, clearly missed that one. I just fixed the method name (now `getIDfromMavenCoordinates()`), squashed and pushed again.